pkg/qtum/method.go

```diff
@@ -206,3 +206,10 @@ func (m *Method) ListUnspent(req *ListUnspentRequest) (resp *ListUnspentResponse
 	}
 	return
 }
+
+func (m *Method) SendRawTransaction(req *SendRawTransactionRequest) (resp *SendRawTransactionResposne, err error) {
```

Comment:
See other comment. This is unneccesary as already implemented in transformer package.